util/start_test

```diff
@@ -1099,6 +1099,12 @@ def set_up_executables():
                         "{0}".format(prediff))
                 finish()
             chpl_system_prediff.append(prediff)
+    if launcher == "slurm-srun":
+        # With Slurm-based launcher, auto-run prediff-for-slurm-srun.
+        pdSlurm = os.path.join(util_dir, "test", "prediff-for-slurm-srun")
+        if not any(pdSlurm in csp for csp in chpl_system_prediff):
```

Comment:
```python
if pdSlurm not in chpl_system_prediff:
```

?

Reply:
Oh, I guess I could do that, yeah.  I started out with just `if not any("prediff-for-slurm-srun" in csp ...` and then switched to looking for the entire path.  But when I switched I didn't remove the no-longer-needed `csp for csp in`.